Apps/frontend/src/components/WaveControls.svelte

```diff
@@ -7,10 +7,48 @@
   import { timeSweep } from "../stores.js";
   import TimeSweepSlider from "./TimeSweepSlider.svelte";
   import ThicknessSwitch from "./ThicknessSwitch.svelte";
+  import DistributeOffsetButton from "./DistributeOffsetButton.svelte";
+  import startStopLine from "./Waves.svelte";
   export let waveElement;
   export let isEnabled;
   export let indicatorElement;
   let btnOnOff;
+
+  let activeChannels = [];
+  let activeChannelCounter = NUM_CHANNELS;
+
+  let distributed = false;
```

Comment:
This is not used anywhere. If we no longer need it, please remove.

Reply:
Is no longer needed (answered in JF). We keep it as stupid as possible and only distribute when we click the button, nothing more, nothing less.

Reply:
Thanks for the hint, i forgot some other old code as well 👍